extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/ZonePlayer.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yet they are discovered, can be volume controlled and so forth. Therefore, they should be exposed as Thing, albeit controlled functionally by a ZoneCoordinator